src/main/runner/entrypoint.ts

```diff
@@ -0,0 +1,14 @@
+// Import shims first to mutate k6 modules globally
+import './shims/http'
+import './shims/browser'
+
+// @ts-expect-error - Path will be replaced at runtime
+// eslint-disable-next-line import/no-unresolved
+import * as userScript from '__USER_SCRIPT_PATH__'
+
+export default function () {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access
+  userScript['default']()
+}
+
```

Comment:
@allansson I made two changes to simplify the implementation:
1. Only the default scenario is supported since we don't have a way to select other scenarios from the UI
2. The `options` export is missing in the entry script, but is added in `intrumentation.ts`. We also don't merge the user's option with the default debug options, but that's in line with the current behavior.